ruby/spec/cucumber/ci_environment/detect_ci_environment_spec.rb

```diff
@@ -21,7 +21,8 @@
 
   context 'with no CI environment' do
     subject { JSON.parse(ci_environment.to_json) }
-    let(:ci_environment) { Cucumber::CiEnvironment.detect_ci_environment(env) }
+
+    let(:ci_environment) { described_class.detect_ci_environment(env) }
```

Comment:
this can be simplified to pass in the empty hash to the let here to remove an excess line